deps/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/deps/transport/amqp/AmqpConnectionTest.java

```diff
@@ -170,28 +170,6 @@ public void OpenThrowExceptionReactor() throws IOException, InterruptedException
 
         //assert
     }
-
-    @Test (expected = IOException.class)
-    public void OpenThrowsOnWaitLock() throws IOException, InterruptedException
-    {
```

Comment:
Very orthogonal, but this unit test takes 1 minute to run, and adds nothing valuable